scripts/circleci/setup-mdc-canary.js

```diff
@@ -1,16 +1,37 @@
 const {join} = require('path');
-const {spawn} = require('child_process');
+const {spawn, spawnSync} = require('child_process');
 const packageJson = require(join(__dirname, '../../package.json'));
+
+// Note that yarn also has a `versions` field, but it appears to be sorting canary versions
+// alphabetically so we can't depend on it. Instead we use `time` which is a version to time map.
+const versionsProcess = spawnSync('yarn', ['info', 'material-components-web', 'time', '--json'], {
```

Comment:
Have you considered just using: ` yarn info material-components-web dist-tags.canary`?

Reply:
I wasn't aware of it, but it's even cleaner 👍